coding/unit_test_coverage.rst

```diff
@@ -13,25 +13,26 @@ executing instrumented unit tests which records the complete execution path
 through the code and then calculating metrics indicative of the coverage
 achieved during execution.
 
-Coverage analysis examines the output of a code instrumented to record every
+Coverage Analysis examines the output of a code instrumented to record every
```

Comment:
"Coverage Analysis" isn't a proper name -- I wouldn't capitalize it. Looks like it's inconsistently handled through this file, but I'd prefer to fix that by down-, rather than up-, casing.